static/css/yucibot.css

```diff
@@ -124,6 +125,8 @@
 }
 
 #searchBar {
```

Comment:
I would suggest to minimize the usage of `#id` selectors. The rules say that all ids on html elements should be unique. This means, that you can never re-use this CSS.

You don't have to change it, but keep it in mind 😉